### PR TITLE
feat!: Only support latest versions of Terraform and Official GitHub Terraform Provider

### PR DESCRIPTION
**Pull request by mariux**

### CODE REVIEW

**Comment by mariux** [comment]
relates to relates to https://github.com/mineiros-io/terraform-github-team/pull/47

**Comment by mariux** [comment]
relates to relates to https://github.com/mineiros-io/terraform-github-repository/pull/74

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,8 +7,28 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+### Added
+
+- Add support for Official GitHub Terraform Provider `integrations/github`
+
+### Removed
+
+- Removed support for Terraform < 1.0
+- Removed support for GitHub Provider < 4.0
+- Removed compatibility to Hashicorp GitHub Terraform Provider `hashicorp/github`
```
Comment (soerenmartius) [inline CHANGELOG.md:26]:
```suggestion
- Removed support for Terraform `< 1.0`
- Removed support for GitHub Provider `< 4.0`
- Removed compatibility to Hashicorp GitHub Terraform Provider `hashicorp/github`
```

**Review by soerenmartius — APPROVED** [review]
lgtm! Good job! Please find the minor comments!

File: CHANGELOG.md (hunk 1 above)
Comment (mariux) [reply CHANGELOG.md:26]: will update before release thanks

**Pull request merged**